README.md

```diff
@@ -47,6 +47,15 @@ And now to generate an ID:
 Optional<Id> id = icicleIdGenerator.generateId();
 ```
 
+Or a batch of IDs;
+
+```java
+Optional<List<Id>> ids = icicleIdGenerator.generateIdBatch(10); // Get up to 10 IDs
+Optional<List<Id>> ids = icicleIdGenerator.generateIdBatch(); // Get up to 4096 IDs
+```
+
+The batch will return a list of IDs up to the size of the specified batch, if no size is specified then it will default to the maximum sequence size. If the batch size is greater than the maximum sequence minus the current sequence in redis, then you will receive a list of IDs, the size of which is the difference between the current and maximum sequence.
```

Comment:
I wonder if it would be better to summarise this as:

> When requesting a batch of IDs, the generator cannot always return you the amount you asked for (this is a limitation related to our strong uniqueness guarantees) - instead, it will return between 1 and `n`, where `n` is the number of IDs you asked for in the batch. You may wish to check how many you actually received, and request more until you have the total you need.

Just a bit less heavy on the references to sequence (shouldn't have to know what this is) and maximum sequence (which they'd have to look up if they didn't know it), etc.